package/MDAnalysis/coordinates/INPCRD.py

```diff
@@ -75,3 +75,10 @@ def _read_first_frame(self):
                 line = inf.readline()
                 for i in range(3):
                     self.ts._pos[-1, i] = float(line[i*12:(i+1)*12])
+
+    @staticmethod
+    def parse_n_atoms(filename, **kwargs):
+        with open(filename, 'r') as f:
+            f.readline()
```

Comment:
I'm guessing the logic here is that you want to read from the second line of the file, but avoid parsing in all lines of the file with i.e., `f.readlines()[1].split()[0]`; I'm assuming your approach is unambiguously faster (?), though this may depend on what happens under the hood with the file object.

Reply:
The Topology file is probably 100s of lines, so I'm expecting/hoping that making two `readline` calls is quicker than slurping the entire thing up and splitting around all newlines etc